### PR TITLE
Editor | Project extent placeholder

### PR DESCRIPTION
**Pull request by mddilley**
This PR adds a placeholder map for the project extent. It can be found in the 3rd step of the New Project stepper. Happy to update with any feedback. There is no interaction with the DB for now. Selected items are temporarily held in state.

#### Project Map
Screen Shot 2020-12-09 at 5 21 39 PM

### CODE REVIEW

File: moped-editor/src/views/projects/mapHelpers.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import React from "react";
+
+export const MAPBOX_TOKEN = `pk.eyJ1Ijoiam9obmNsYXJ5IiwiYSI6ImNraWV4dHR0ZjAwNnYyd3FwYjFoNHduaDcifQ.--3vRm2KHq1gh5K_L0pqtA`;
```
Comment (sergiogcx) [inline moped-editor/src/views/projects/mapHelpers.js:3]: This should be part of the environment variables I think

**Comment by sergiogcx** [comment]
Woah! Impressive work!

**Review by sergiogcx — APPROVED** [review]
Amazing work as always!

**Review by mateoclarke — APPROVED** [review]
Looks great Mike! 

teigen_clapping

File: moped-editor/src/views/projects/mapHelpers.js (hunk 1 above)
Comment (mddilley) [reply moped-editor/src/views/projects/mapHelpers.js:4]: Cool, it is restricted to staging, production, and dev for now but I think that y'all are right that we should hide it away. I'll create an issue to delete this token, create a fresh one, and update where this comes from. 👍

**Pull request merged**